client/src/app/routes/Visualizer.scss

```diff
@@ -22,7 +23,7 @@
     justify-content: stretch;
     height: 70vh;
     overflow: hidden;
-    border: 1px solid $gray-light;
+    border: 1px solid var(--input-border-color);
     border-radius: 6px;
 
     .viva {
```

Comment:
Replace .viva background rule from #ffffff to appropriate variable on line 35 and remove rule starting on line 51: `&.dark-mode {
    .graph-border {
      .viva {
        background: $gray-midnight;
      }
    }
  }
`

Reply:
Resolved